frontend/src/metabase/components/TokenField.jsx

```diff
@@ -383,19 +389,19 @@ export default class TokenField extends Component {
 
         const valuesList =
           <ul
-              className={cx("m1 px1 pb1 bordered rounded flex flex-wrap bg-white", {
-                [`input--focus border-${color}`]: this.state.focused
+              className={cx("m1 p0 bordered rounded flex flex-wrap bg-white", {inputBoxClasses}, {
```

Comment:
I don't think you want the brackets around `inputBoxClasses`. That's going to set a class called "inputBoxClasses" to the value of inputBoxClasses